docs/SimpleFormIterator.md

```diff
@@ -156,9 +156,9 @@ import { ArrayInput, SimpleFormIterator, DateInput, TextField, FormDataConsumer,
     <SimpleFormIterator disableRemove>
         <DateInput source="date" />
         <FormDataConsumer>
-            {({ getSource }) => (
+            {({ scopedFormData }) => (
                 <Labeled label="Url">
-                    <TextField source={getSource('url')} />
+                    <TextField source="url" record={scopedFormData} />
```

Comment:
Are you sure about the removal of `getSource` ?

Reply:
Yes, this solution was tested in the context of this PR: https://github.com/marmelab/react-admin/pull/8359 (have a look at the comments)

Reply:
Does it mean we don't need `getSource` anymore?

Reply:
we still need it for inputs